src/api/SetTokenAPI.ts

```diff
@@ -22,10 +22,10 @@ import { Address } from 'set-protocol-utils';
 import { SetTokenContract, VaultContract } from 'set-protocol-contracts';
 
 import { ZERO } from '../constants';
-import { coreAPIErrors } from '../errors';
+import { coreAPIErrors, setTokenAssertionsErrors } from '../errors';
```

Comment:
TODO: some inconsistent naming.
`coreErrors` file name -> `coreAPIErrors` class
`setTokenErrors` file name -> ``setTokenAssertionsErrors` class

Reply:
let's make sure to address this in separate pr

Reply:
+1 Added a trello card